app.py

```diff
@@ -29,6 +30,34 @@
 
     '''),
 
+    dcc.DatePickerSingle(
+        id='date-picker-single',
+        initial_visible_month=dt.datetime(1997, 5, 5),
+        min_date_range=dt.datetime(1997, 4, 29),
```

Comment:
although we can actually just remove this example all together. I think that I commited it accidently.